### PR TITLE
chore: upgrade TC in android app readme

### PR DESCRIPTION
**Pull request by parkjh**

### CODE REVIEW

**Comment by parkjh** [comment]
@aronshamash see https://github.com/newsuk/nu-projectd-times-smartphone-android/pull/704

File: android-app/README.md [hunk 1]
```diff
@@ -62,3 +62,7 @@ uploaded to maven.
 4. Copy newly built android bundle to andriod repo `cp android-app/xnative/src/main/assets/index.android.bundle ../../nu-projectd-times-smartphone-android/mobile/src/main/assets/index.android.bundle`
 5. before building, update the version in android-app/package.json to something that doesnt exist in nu-android, mobile/build.gradle change the version to match in the top level build.gradle add the local repo: `maven { url "../../times-components/android-app/repo"}`
 6. then using android studio build the app onto the device
+
+### Upgrade Times Component in the Android app
+
+see [Readme](https://github.com/newsuk/nu-projectd-times-smartphone-android/blob/develop/CONTRIBUTING.md)
```
Comment (ValGeorgiev) [inline android-app/README.md]: Could we have more description, this is not helping at all

Comment (ValGeorgiev) [reply android-app/README.md]: Sorry i didn't saw your comment, but could we change it to navigate to this section directly

Comment (parkjh) [reply android-app/README.md]: @ValGeorgiev it was decided that the exact step should really live in the app repo itself so I only link the document from here...

Comment (ValGeorgiev) [reply android-app/README.md]: I mean that the link should be: `https://github.com/newsuk/nu-projectd-times-smartphone-android/blob/develop/CONTRIBUTING.md#upgrading-to-the-latest-times-component`

**Pull request merged**